### PR TITLE
browserstack-sdk robot

### PR DESCRIPTION
**Pull request by adarshTS**

### CODE REVIEW

File: .pabotsuitenames [hunk 1]
```diff
@@ -0,0 +1,9 @@
+datasources:c36e329fd2e134d2bc6e5863265be76be884280f
+commandlineoptions:97d170e1550eee4afc0af065b78cda302a97674c
+suitesfrom:no-suites-from-option
+file:21ab1486120a36fe5209ba652a3a5ba39c74bb8e
+--suite Suites.E2E.E2E
+--suite Suites.Login.loginTests
+--suite Suites.Offers.Offers Test
+--suite Suites.Product.Filter Test
+--suite Suites.User.User Tests
```
Comment (princebaretto99) [inline .pabotsuitenames]: Remove this file

File: README.md [hunk 2]
```diff
@@ -9,35 +9,33 @@ Robot Framework has easy syntax, utilising human-readable keywords and its capab
...
-- Install the webdriver-framework from the cloned repository locally
-    ```sh
-    cd selenium
-    pip install wheel
-    pip install .
-    ```
-- Clone the current repository
-    ```sh
-    git clone https://github.com/browserstack/browserstack-examples-robot.git
+    git checkout sdk-develop
```
Comment (princebaretto99) [inline README.md]: Replace with main

File: README.md [hunk 3]
```diff
@@ -60,160 +58,42 @@ The Selenium test tests are run on different platforms like on-prem, docker and
...
+### Run a specific test
 
 - How to run the test?
 
-  To run a specific test scenario, use the following command with the additional 'test-name' argument:
+  To run a specific test scenario, use the command below:
   
   ```sh
-  robot --variable testType:on-prem --test "<Test scenario name>" . 
+  browserstack-sdk robot /path/to/test.robot .
```
Comment (princebaretto99) [inline README.md]: Fix this

File: README.md [hunk 4]
```diff
@@ -60,160 +58,42 @@ The Selenium test tests are run on different platforms like on-prem, docker and
...
 
-
-  where,  the argument 'test-name' can be any RobotFramework scenario name configured in this repository.
-  
-  E.g. "Check Login with locked_user", "E2E Test", "Apply 'Lowest to Highest' Order By Filter" or any of the other test scenario names, as outlined in [About the tests in this repository](#About-the-tests-in-this-repository) section.
-
 - Output
 
-  This run profile executes a specific test scenario on a single browser instance on your own machine.
+  This run profile executes the specific test scenario in browserstack on the browsers configured in your `browserstack.yml` file.
```
Comment (princebaretto99) [inline README.md]: replace with PLatforms

File: README.md [hunk 5]
```diff
@@ -60,160 +58,42 @@ The Selenium test tests are run on different platforms like on-prem, docker and
...
+### Run the entire test suite
 
 - How to run the test?
 
-  To run the entire test suite on your own machine, use the following command:
+  To run the entire test suite parallely in browserstack, you will require [pabot dependency](https://pabot.org/)  use the following command:
   
   ```sh
-  robot --variable testType:on-prem --suite <Suite-name> .
+  browserstack-sdk pabot path/to/testsuiteFolder .
```
Comment (princebaretto99) [inline README.md]: Fix this

File: README.md [hunk 6]
```diff
@@ -60,160 +58,42 @@ The Selenium test tests are run on different platforms like on-prem, docker and
...
   ```
-
+  You can also use the other combinations as described in [pabot](https://pabot.org/) to run your tests parallely. 
 
 - Output
 
-  This run profile executes the entire test suite sequentially on a single browser, on your own machine.
+  This run profile executes the entire test suite parallely in browserstack on the browsers configured in your `browserstack.yml` file.
```
Comment (princebaretto99) [inline README.md]: PLatforms

File: resources/conf/runners/Library.py [hunk 7]
```diff
@@ -88,19 +69,6 @@ def set_loc(lat, long):
...
-        se2lib.driver.execute_script(
-            'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"failed","reason": "Oops! my sample test failed"}}')
-
-
-def set_session_name(new_name):
-    global se2lib
-    se2lib.driver.execute_script(
-        'browserstack_executor: {"action": "setSessionName", "arguments": {"name": "' + new_name + '"}}')
+def add_wait():
+    import time
```
Comment (princebaretto99) [inline resources/conf/runners/Library.py]: Shift to first line

**Review by princebaretto99 — CHANGES_REQUESTED** [review]

**Review by princebaretto99 — DISMISSED** [review]

File: resources/conf/runners/keywords.robot [hunk 8]
```diff
@@ -1,81 +1,28 @@
...
-
-
-Stop Test
-    [Arguments]    ${testType}
-
-    Only Run On Browserstack    Mark Test Status                 ${TEST_STATUS}
-    Close Browser
+Start Onprem Test
+    # Create Webdriver    Chrome    executable_path=/Users/adarsh/Downloads/chromedriver
+    # Go To    http://bstackdemo.com/
```
Comment (samirans89) [inline resources/conf/runners/keywords.robot]: needed or can be deleted?

Comment (adarshTS) [reply resources/conf/runners/keywords.robot]: Can be deleted, removed the comments.

**Review by princebaretto99 — APPROVED** [review]

**Pull request merged**